spec/coder_spec.lua

```diff
@@ -1608,44 +1609,54 @@ describe("Pallene coder /", function()
 
             typealias y = integer
 
+            local a, a = 1, 2
+
             ------
 
-            function local_type(): integer
+            export function local_a(): integer
```

Comment:
In the other tests next to this we are using the name "local variable" to refer to local variables inside a function. It might be better to mention "toplevel variable" instead.